### ISSUE TITLE
Estimate the probability that ESN learns laminarization (Ref 1, Q3)

### ISSUE THREAD

**Issue by anton-pershin** [anton-pershin/restools#3]
Referee 1 comment:
> 3. In Fig. 1, the authors demonstrated that the trained ESN with data from
turbulent regime at various Reynolds numbers can qualitatively predict the relax
to laminar regime at a different Reynolds number. However, it is difficult to
believe trained ESN would always successfully predict the relax to laminar
regime, no matter how to change the settings in training (e.g., training sets,
validation set, random seed). If the chance was close to 50/50, the conclusion
would be invalid. The reviewer suggests training ensemble of ESNs with different
settings and checking if majority of those ESNs would predict the relax to
laminar regime.

**Comment by anton-pershin** [comment]
Started playing with Re = 275. Original tasks (true model) are 42, 45 (each corresponds to 100 ICs). Using launcher_of_ESN_ensemble.

First test run -- 10 ICs and 10 ESNs, spectral_radius = 0.5, sparsity = 0.5 (task 92).

**Comment by anton-pershin** [comment]
It seems everything works as expected. Created new view: lifetime_distributions_ensemble_esn. Use it for test plotting

Second test run -- 200 ICs and 10 ESNs, spectral_radius = 0.5, sparsity = 0.5 (task 93). If ok, maybe change sparsity to 0.9 (was found optimal) and run 100 ESNs

**Comment by anton-pershin** [comment]
Second test run is ok. Need to changed sparsity to 0.9 and run again 200 ICs and 10 ESNs. If ok, run more 9 identical tasks

Found a server-related problem. We have limited HDD memory there => task 93 is actually broken because many ESNs failed to save their predictions. I am freeing the memory now

**Comment by anton-pershin** [comment]
Fixed this. Just run second test again but with sparsity 0.9

**Comment by anton-pershin** [comment]
It is ok. Now need to run more 9 identical tasks. Also remember that I need to update random_seed_starts_at. It was 1 to 10 for second run, now it will be 11 to 20, 21 to 30, ... , 91 to 100

Run for 11 to 20.

**Comment by anton-pershin** [comment]
Finished second task (11 to 20)

Run 3rd task (21 to 30)

**Comment by anton-pershin** [comment]
Finished 3rd task (21 to 30)

Run 4th task (31 to 40)

**Comment by anton-pershin** [comment]
Finished 4th task

Run 5th task (41 to 50)

**Comment by anton-pershin** [comment]
Finished 5th task

Run 6th task (51 to 60)

**Comment by anton-pershin** [comment]
Finished 6th task

Run 7th task (61 to 70)

**Comment by anton-pershin** [comment]
Finished 7th task

Run 8th task (71 to 80)

**Comment by anton-pershin** [comment]
Finished 8th task

Run 9th task (81 to 90)

**Comment by anton-pershin** [comment]
Finished 9th task

Run 10th task (91 to 100)
